### DISCUSSION TITLE
How to query Server Stats?

### DISCUSSION THREAD

**Discussion by TheIdealMatrix** [Minestom/Minestom#1860] [Help]
Anyone who can tell me how to set up the Query System?
I want to query Server Information using a Website like this one: https://mcsrvstat.us/.

I've found this Class: https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/extras/query/Query.java
but it does not seem to work as expected. I tried to query it with an PHP Script, which works 100% on normal servers, but not on Minestom.

Also, is the stop Method missing the static attribute, or how am I supposed to stop the system when the server stops?

**Comment by KrystilizeNevaDies** [comment]
Static issue is fixed:
https://github.com/Minestom/Minestom/pull/1867
